media-gfx/graphicsmagick/graphicsmagick-1.3.35.ebuild

```diff
@@ -0,0 +1,132 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit autotools toolchain-funcs
+
+MY_P=${P/graphicsm/GraphicsM}
+
+DESCRIPTION="Collection of tools and libraries for many image formats"
+HOMEPAGE="http://www.graphicsmagick.org/"
+LICENSE="MIT"
+SLOT="0/${PV%.*}"
+
+if [[ ${PV} == "9999" ]] ; then
+	inherit mercurial
+	EHG_REPO_URI="http://hg.code.sf.net/p/${PN}/code"
```

Comment:
$ curl --head http://hg.code.sf.net
HTTP/1.1 301 Moved Permanently
Server: nginx/1.14.0 (Ubuntu)
Date: Wed, 25 Mar 2020 10:11:27 GMT
Content-Type: text/html; charset=iso-8859-1
Content-Length: 232
Connection: keep-alive
Location: https://sourceforge.net/

Reply:
The actual repo URI doesn't have a redirect and is still linked from upstream's pages.